heatmap_function.R

```diff
@@ -0,0 +1,90 @@
+library(shiny)
+library(shinydashboard)
+library(leaflet)
+library(leaflet.extras)
+library(sf)
+library(ggplot2)
+library(dplyr)
+library(lubridate)
```

Comment:
All libraries imported in app.R do not need to be re-imported here.

Reply:
It looks like the smaller the interval you have, the less flashing there is...